frontend/src/SignUp.css

```diff
@@ -0,0 +1,58 @@
+body {
+    display: flex;
```

Comment:
instead of using pixels e.g border: 2px

use % or enum because pixel CSS is screen dependent